src/OrchardCore/OrchardCore.Recipes.Abstractions/Services/IRecipeEnvironmentProvider.cs

```diff
@@ -0,0 +1,10 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace OrchardCore.Recipes.Services
+{
+    public interface IRecipeEnvironmentProvider
+    {
+        Task SetEnvironmentAsync(IDictionary<string, object> environment);
```

Comment:
Maybe `PopulateEnvironmentAsync` or `MergeEnvironmentAsync`, but not sure, too tired ;)

Reply:
I did the change with `PopulateEnvironmentAsync()`

Revert it if you don't like it, i'm also fine with `SetEnvironmentAsync()` ;)

Reply:
I like `PopulateEnvironmentAsync` thanks.